package.json

```diff
@@ -25,6 +25,7 @@
   "devDependencies": {
     "babel-eslint": "6.0.0-beta.6",
     "coveralls": "2.11.8",
+    "doctrine": "^1.2.0",
```

Comment:
Whoops! Quite right.
On 25 Mar 2016 01:46, "Jordan Harband" <notifications@github.com> wrote:

> In package.json
> <https://github.com/yannickcr/eslint-plugin-react/pull/513#discussion_r57411882>
> :
>
> > @@ -25,6 +25,7 @@
> >    "devDependencies": {
> >      "babel-eslint": "6.0.0-beta.6",
> >      "coveralls": "2.11.8",
> > +    "doctrine": "^1.2.0",
>
> this would need to be a full dependency since it's not related to
> actually developing the module.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yannickcr/eslint-plugin-react/pull/513/files/f3e634c59e8928ab170a9bfa9d056cf0f8778064#r57411882>
>